wsgi/odyssy/announcement/models.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import unicode_literals
```

Comment:
Add `staticmethod` for whatever queries are required. This will make things easier later on.